test/parallel/test-http-server-keep-alive-timeout.js

```diff
@@ -49,13 +43,7 @@ test(function serverEndKeepAliveTimeoutWithPipeline(cb) {
 });
 
 test(function serverNoEndKeepAliveTimeoutWithPipeline(cb) {
-  let requestCount = 0;
-  process.on('exit', () => {
-    assert.strictEqual(requestCount, 3);
-  });
-  const server = http.createServer((req, res) => {
-    requestCount++;
-  });
+  const server = http.createServer(common.mustCall((req, res) => {}, 3));
```

Comment:
Since the function inside the `mustCall()` is a non-op, this can be shortened to just:

```js
const server = http.createServer(common.mustCall(), 3));
```